src/constants/apiPaths.ts

```diff
@@ -1,6 +1,6 @@
 
 const API_PATHS = {
-  product: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
+  product: 'https://tytd8guixe.execute-api.eu-west-1.amazonaws.com/dev',
   order: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
   import: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
   bff: 'https://.execute-api.eu-west-1.amazonaws.com/dev'
```

Comment:
I suggest to change this URL too, because it's used to fetch products in other components